applications/dashboard/controllers/api/MediaApiController.php

```diff
@@ -16,6 +16,8 @@
  * API Controller for `/media`.
  */
 class MediaApiController extends AbstractApiController {
+    const TYPE_IMAGE = 'image';
+    const TYPE_ANY = 'any';
```

Comment:
What do you think about changing "any" to "other"? The word "any" would, literally, include images. The word "other" would (hopefully) convey the user is choosing to upload something that doesn't match our existing types.

Additionally, I liked your previous implementation of specific type-based "buckets" of extensions. It would let us to grow specific types, allowing for custom input/output handling, while making users be more explicit in what they're sending. It'd also be easier to expand on those types, instead of having to limit the scope of an umbrella "any"/"other" type, later.

Ultimately, it's your decision!